libraries/models.py

```diff
@@ -406,6 +397,21 @@ def library_repo_url_for_version(self):
     def get_release_notes(self):
         return get_release_notes_for_library_version(self)
 
+    def get_cpp_standard_minimum_display(self):
+        """Returns the display name for the C++ standard, or the value if not found.
+
+        Source of values is
+        https://docs.cppalliance.org/user-guide/prev/library_metadata.html"""
+        display_names = {
+            "98": "C++98",
+            "03": "C++03",
+            "11": "C++11",
+            "14": "C++14",
+            "17": "C++17",
+            "20": "C++20",
+        }
+        return display_names.get(self.cpp_standard_minimum, self.cpp_standard_minimum)
```

Comment:
Yeah, this method is a bit of an odd one to me, it's not used at the moment and I considered removing it but figured  it could be useful if it were changed to provide the text as used in the libraries pages too, which contains spans. It's something to think about later.